release/cloudbuild.yaml

```diff
@@ -54,22 +58,22 @@ steps:
   env:
   - GIT_TAG=${_GIT_TAG}
   - PROJECT_ID=${PROJECT_ID}
+  - KEY_LOCATION=${_KEY_LOCATION}
+  - KEY_RING=${_KEY_RING}
+  - KEY_NAME=${_KEY_NAME}
+  - KEY_VERSION=${_KEY_VERSION}
   args:
   - '-c'
   - |
     apt-get update
     apt-get install libpcsclite-dev -y
     cp ./dist/cosign-linux_linux_amd64/cosign_linux_amd64 /usr/local/bin/cosign
     make sign-container-cloudbuild
-  secretEnv:
-  - COSIGN_PASSWORD
 
 availableSecrets:
   secretManager:
   - versionName: projects/${PROJECT_NUMBER}/secrets/GITHUB_TOKEN/versions/latest
```

Comment:
`${GITHUB_TOKEN}`?

Reply:
PROJECT_ID should also work for ${PROJECT_NUMBER}

Reply:
GITHUB_TOKEN is the name of the secret Key i've created in the KMS side

Reply:
can I change that in a follow up? 

> PROJECT_ID should also work for ${PROJECT_NUMBER}

Reply:
sure